{"name": "创建英雄失败的测试用例-攻击力不符合要求", "status": "passed", "parameters": [{"name": "success_for_name", "value": "'807834167'"}, {"name": "success_for_volume", "value": "99"}, {"name": "fail_for_power", "value": "-50"}], "start": 1684150563175, "stop": 1684150563175, "uuid": "b13fa556-0ff7-4964-b986-9f0ebe2a85af", "historyId": "c0dd98e3fca60f3fa32db28ebc6b6cc4", "testCaseId": "052da0c40443119f715fb4201d430dc8", "fullName": "test_create_hero_homework_01.TestHeroCreate#test_create_hero_fail_for_power", "labels": [{"name": "tag", "value": "run(order=4)"}, {"name": "suite", "value": "test_create_hero_homework_01"}, {"name": "subSuite", "value": "TestHeroCreate"}, {"name": "host", "value": "LAPTOP-0EHG7NSH"}, {"name": "thread", "value": "48660-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_create_hero_homework_01"}]}